src/main/java/nl/mpcjanssen/simpletask/util/Util.kt

```diff
@@ -72,12 +72,28 @@ fun getString (resId : Int) : String {
     return TodoApplication.app.getString(resId)
 }
 
-fun showConfirmationDialog(cxt: Context, msgid: Int,
-                           okListener: DialogInterface.OnClickListener, titleid: Int) {
-    val show = Config.showConfirmationDialogs
-
+fun showConfirmationDialog(cxt:        Context,
+                           msgid:      Int,
+                           okListener: DialogInterface.OnClickListener,
+                           titleid:    Int) {
     val builder = AlertDialog.Builder(cxt)
     builder.setTitle(titleid)
+    showConfirmationDialog(msgid, okListener, builder)
+}
+
+fun showConfirmationDialog(cxt:        Context,
+                           msgid:      Int,
+                           okListener: DialogInterface.OnClickListener,
+                           title:      CharSequence) {
+    val builder = AlertDialog.Builder(cxt)
+    builder.setTitle(title)
+    showConfirmationDialog(msgid, okListener, builder)
+}
+
+private fun showConfirmationDialog(msgid:      Int,
+                           okListener: DialogInterface.OnClickListener,
+                           builder:    AlertDialog.Builder) {
+    val show = Config.showConfirmationDialogs
```

Comment:
So, builder.setTitle can take either a string or a stringId. We'd been passing a stringId, but that doesn't make sense in the context of confirmation dialogs, so I overloaded the function so you can call it with either a string or an Id.